src/index.tsx

```diff
@@ -250,26 +267,28 @@ const AuthIsLoaded: React.FunctionComponent<React.PropsWithChildren<{}>> = ({
   return <>{children}</>;
 };
 
-render(
-  <BugsnagErrorBoundary>
-    <ThemeProvider theme={theme}>
-      <Elements stripe={stripePromise}>
-        <DndProvider backend={HTML5Backend}>
-          <Provider store={store}>
-            <ReactReduxFirebaseProvider {...rrfProps}>
-              <CustomSoundsProvider>
-                <AuthIsLoaded>
-                  <AppRouter />
-                </AuthIsLoaded>
-              </CustomSoundsProvider>
-            </ReactReduxFirebaseProvider>
-          </Provider>
-        </DndProvider>
-      </Elements>
-    </ThemeProvider>
-  </BugsnagErrorBoundary>,
-  document.getElementById("root")
-);
+traceReactScheduler("initial render", performance.now(), () => {
+  render(
+    <BugsnagErrorBoundary>
+      <ThemeProvider theme={theme}>
+        <Elements stripe={stripePromise}>
+          <DndProvider backend={HTML5Backend}>
+            <Provider store={store}>
+              <ReactReduxFirebaseProvider {...rrfProps}>
+                <CustomSoundsProvider>
+                  <AuthIsLoaded>
+                    <AppRouter />
+                  </AuthIsLoaded>
+                </CustomSoundsProvider>
+              </ReactReduxFirebaseProvider>
+            </Provider>
+          </DndProvider>
+        </Elements>
+      </ThemeProvider>
+    </BugsnagErrorBoundary>,
+    document.getElementById("root")
+  );
+});
```

Comment:
I wonder if measuring the performance can affect the performance in the prod

Reply:
Technically yes, slightly, since we have profiling enabled in production (if I remember correctly)

Though IMO getting the data is far more useful than the tiny performance hit it will add.